catalog/transformer/catalog-transformer-geojson-input/src/main/java/ddf/catalog/transformer/input/geojson/GeoJsonInputTransformer.java

```diff
@@ -183,12 +163,39 @@ public Metacard transform(InputStream input, String id)
     }
 
     if (id != null) {
-      metacard.setId(id);
+      metacard.setAttribute(new AttributeImpl(Metacard.ID, id));
     }
 
     return metacard;
   }
 
+  public Metacard getBaseMetacard(Map<String, Object> properties, List<MetacardType> metacardTypes)
```

Comment:
❓ Should it be protected if it is only required by subclasses or private if no one is expected to override it?

Reply:
Agreed - updated.